rosdep/base.yaml

```
@@ -3088,15 +3088,21 @@ libdc1394-dev:
   arch:
     pacman:
       packages: [libdc1394]
-  debian: [libdc1394-22-dev]
+  debian:
+    '*': [libdc1394-dev]
+    'buster': [libdc1394-22-dev]
+    'stretch': [libdc1394-22-dev]
```

While we aren't totally consistent about this, we don't typically quote these:

```suggestion
    buster: [libdc1394-22-dev]
    stretch: [libdc1394-22-dev]
```